docs/libs/gelly_guide.md

```diff
@@ -240,6 +240,7 @@ Graph<Long, Double, Double> networkWithWeights = network.joinWithEdgesOnSource(v
     <img alt="Union Transformation" width="50%" src="fig/gelly-union.png"/>
 </p>
 
+* <strong>Difference</strong>: Gelly's `difference()` method performs a difference on the vertex and edge sets of the input graphs. The resultant graph is formed by removing the vertices and edges from the graph that are common with the second graph.
```

Comment:
we can rephrase this a bit.. there is one input graph and no "second" graph... I guess you copied from the union description above (which should also be changed).